vtex-apps/tiktok-tbp-callback
# Feature/brpa 750 add redirect middleware

#### What is the purpose of this pull request?

- Add redirect middleware for redirecting to proper account.
- Increase timeout to 2 seconds.
- Update dependencies
- Refactor: turn URLState changes more scalable.
  - Avoid the duplication of the parameters types.

#### How to test it?

You can see the endpoint working [here](https://tiktoktbp--sandboxbrdev.myvtex.com/v0/tiktok-tbp-callback/redirect?authCode=12345&state={"account":"sandboxbrdev","workspace":"master","salesChannelId":1,"passPhrase":"abc"}). It will be redirect to `https://sandboxbrdev.myvtex.com/v0/tiktok-tbp/connect?authCode=12345&salesChannelId=1&passPhrase=abc`

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/EIRNuZ1p9TI3u/giphy.gif)

#### Types of changes
